apps/go-mod-validator/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from "fs";
+import { getSmartcontractkitDependencies } from "./parser";
+import { validateDependency } from "./github";
+
+async function run() {
+  // Access command-line arguments
+  const args = process.argv.slice(2);
+  if (args.length == 0) {
+    console.error("need 1 argument as the input");
+    process.exit(1);
+  }
+  const goListOutputFile = args[0];
+
+  // Read the file content
+  let goListOutputContent = "";
+  try {
+    goListOutputContent = fs.readFileSync(goListOutputFile, "utf-8");
+  } catch (err) {
+    console.error(
+      `failed to read input file: ${goListOutputFile}, with err: ${err}`,
+    );
+    process.exit(1);
```

Comment:
Example of `@actions/core` usage instead:

- `core.error("...")`
- `core.setFailed("...")`

Reply:
Sure, I will make those changes in this [ticket](https://smartcontract-it.atlassian.net/browse/RE-2786)